src/features/virtualDocumentTracker.ts

```diff
@@ -50,7 +50,7 @@ function trackFutureVirtualDocuments(server: OmniSharpServer, eventStream: Event
 }
 
 function shouldIgnoreDocument(document: TextDocument, server: OmniSharpServer): boolean {
-    if (document.uri.scheme === 'file' || document.languageId !== 'csharp') {
+    if (document.uri.scheme === 'file' || document.languageId !== 'csharp' || document.uri.scheme !== "virtualCSharp") {
```

Comment:
Could this be simplified to `document.uri.scheme !== "virtualCSharp" || document.languageId !== 'csharp'